experimenter/experimenter/nimbus-ui/src/components/Summary/SummaryTimeline/index.test.tsx

```diff
@@ -26,15 +26,15 @@ describe("SummaryTimeline", () => {
     ].forEach((set) => {
       render(<Subject {...set} />);
 
-      expect(screen.queryByTestId("label-not-launched")).toBeInTheDocument();
-      expect(screen.queryByTestId("label-start-date")).not.toBeInTheDocument();
-      expect(screen.queryByTestId("label-release-date")).toBeNull();
+      expect(screen.getByTestId("label-not-launched")).toBeInTheDocument();
+      expect(screen.getByTestId("label-start-date")).not.toBeInTheDocument();
+      expect(screen.getByTestId("label-release-date")).toBeNull();
```

Comment:
This is incorrect. getByTestId throws if the element is not found and we are asserting this element does not exist. Revert this change